### PR TITLE
[FLINK-7419] Shade jackson dependency in flink-avro

### PR DESCRIPTION
**Pull request by zjureel**
## What is the purpose of the change

Shade jackson dependency in flink-avro to avoid incompatible versions

## Brief change log

  - *Shade jackson in pom.xml of flink-avro project*

## Verifying this change

*(Please pick either of the following options)*

No test case

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (yes)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (no)
  - The serializers: (no)
  - The runtime per-record code paths (performance sensitive): (no)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: (no)

## Documentation

  - Does this pull request introduce a new feature? (no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

File: flink-connectors/flink-avro/pom.xml [hunk 1]
```diff
@@ -174,6 +174,35 @@ under the License.
...
+							<promoteTransitiveDependencies>true</promoteTransitiveDependencies>
+							<artifactSet>
+								<includes>
+									<include>org.codehaus.jackson:*</include>
+								</includes>
+							</artifactSet>
+							<relocations>
+								<relocation>
+									<pattern>org.codehaus.jackson</pattern>
+									<shadedPattern>org.apache.flink.shaded.avro.org.codehouse.jackson</shadedPattern>
```
Comment (zentol) [inline flink-connectors/flink-avro/pom.xml]: Please switch the pattern to `... avro.shaded....` . I would like to reserver the `org.apache.flink.shaded` namespaces for dependencies from flink-shaded.

**Comment by zjureel** [comment]
@zentol Thank you for your review, I have rename the pattern, thanks

**Comment by StephanEwen** [comment]
Merging this...

**Comment by zentol** [comment]
@zjureel The PR has been merged, could you close the PR?

**Pull request closed without merging**